save-backend/src/main/kotlin/org/cqfn/save/backend/scheduling/StandardSuitesUpdateScheduler.kt

```diff
@@ -39,7 +39,7 @@ class UpdateJob(
  * A component that is capable of scheduling [UpdateJob]
  */
 @Service
-@Profile("prod")
+@Profile("automatic-updates")
```

Comment:
I opened #507 to support this more properly, but it can be merged separately.

Reply:
yes, in a separate PR